seleniumapis/vanguard/vanguard.py

```diff
@@ -85,48 +85,49 @@ def get_total_assets(self):
 
-        holdings_table = self.browser.find_element_by_id("BHForm2:accountID:0:_id245")
-        rows = holdings_table.find_elements_by_tag_name("tr")
-
-        # First row is Tables Headers
-        # Second row simply says "ETFs"
-        rows = rows[2:]
-
-        # Row before last simply says "Buy Sell"
-        # Last row is Total Assets
-        rows = rows[:-2]
-
-        # Remaining rows are actual ETFs
-        holdings_info = []
-        for row in rows:
-            holding_info = {}
-            els = row.find_elements_by_tag_name("td")
-
-            # Symbol
-            holding_info["symbol"] = els[0].text
-
-            # Name (needs to be trimmed of spaces)
-            name = els[1].text
-            holding_info["name"] = name.strip()
-
-            # Expense Ratio (Convert to float ratio)
-            holding_info["expense_ratio"] = Converters.percent(els[2].text)
-
-            # Quantity (Convert to doller value)
-            holding_info["quantity"] = Converters.dollar_amount(els[3].text)
-
-            # Last Price (Convert to doller value)
-            holding_info["last_price"] = Converters.dollar_amount(els[4].text)
-
-            # Absolute Change Amount (Convert to doller value)
-            holding_info["change_amount"] = Converters.dollar_amount(els[5].text)
-
-            # Precentage Change (Convert to float ratio)
-            holding_info["change_percent"] = Converters.percent(els[6].text)
-
-            # Current Balance for this holding (Convert to dollar value)
-            holding_info["current_balance"] = Converters.dollar_amount(els[7].text)
-
-            holdings_info.append(holding_info)
-
-        return holdings_info
+        holdings_tables = self.browser.find_elements_by_xpath("//table[starts-with(@id,'BHForm2:accountID:')]")
+        accounts = {}
+        account_num = 0 
+        for holdings_table in holdings_tables:
+            if  len( holdings_table.find_elements_by_tag_name("th") ) == 0:
+                continue
+            else:
+                data_names = ["symbol","name","expense_ratio","quantity","last_price","change_amount","change_percent","current_balance"]
+
+            rows = holdings_table.find_elements_by_tag_name("tr")
+
+            # First row is Tables Headers
+            # Second row simply says "ETFs"
+            rows = rows[2:]
+
+            # Row before last simply says "Buy Sell"
+            # Last row is Total Assets
+            rows = rows[:-2]
+
+            # Remaining rows are actual ETFs
+            holdings_info = []
+            for row in rows:
+                holding_info = {}
+                els = row.find_elements_by_tag_name("td")
+                if len(els) < 8:
+                    continue
+ 
+                for el_num in xrange( len(els) ):
+                    if el_num >= len( data_names ):
+                        continue
+                    holding_info[ data_names[el_num] ] = self._cleanText( els[el_num].text ).strip()
```

Comment:
Previously, the fields of `holding_info` would be parsed based on their type (dollar amount, percentage, etc). This change seems to make it so that they retain their string values as it appears in the Vanguard dashboard. I don't mind that, but wanted to clarify in case that was your intention.